packages/keyboard-shortcuts/src/components/shortcut-provider.tsx

```diff
@@ -1,3 +1,8 @@
+/**
+ * External dependencies
+ */
+import type { JSX } from 'react';
```

Comment:
Can we please directly use `React.JSX` without this import here like we have used `React.HTMLAttributes` below?